### PR TITLE
Do not crash with error 500 when Base64 in an object is malformed.

### PR DESCRIPTION
**Pull request by lolepezy**
Also, refactor error processing to have more sane responses.

### CODE REVIEW

File: src/main/scala/net/ripe/rpki/publicationserver/PublicationService.scala [hunk 1]
```diff
@@ -19,8 +18,9 @@ import net.ripe.rpki.publicationserver.parsing.PublicationMessageParser
 import scala.concurrent.duration._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.io.{BufferedSource, Source}
-import scala.util.{Failure, Success}
+import scala.util.{Failure, Success, Try}
 import akka.util.ByteString
+import com.ctc.wstx.exc.WstxValidationException
```
Comment (ties) [inline src/main/scala/net/ripe/rpki/publicationserver/PublicationService.scala:23]: Is this used?

Comment (lolepezy) [reply src/main/scala/net/ripe/rpki/publicationserver/PublicationService.scala]: No! Removed, thanks.

File: src/main/scala/net/ripe/rpki/publicationserver/PublicationService.scala [hunk 2]
```diff
@@ -69,10 +68,32 @@ class PublicationService
...
+                      Future {
+                        logger.warn("Error while handling request: {}", msgError)
+                        ErrorMsg(msgError)
+                      }
+                  }
+                  r.mapTo[Msg]
+                } catch {
+                  case e: Exception =>
+                    Future {
+                      ErrorMsg(BaseError(BaseError.XmlSchemaValidationError, s"XML parsing/validation error: " + e.getMessage))
```
Comment (ties) [inline src/main/scala/net/ripe/rpki/publicationserver/PublicationService.scala:87]: Is this equivalent?
```suggestion
                      ErrorMsg(BaseError(BaseError.XmlSchemaValidationError, s"XML parsing/validation error: ${e.getMessage}"))
```

Comment (lolepezy) [reply src/main/scala/net/ripe/rpki/publicationserver/PublicationService.scala:87]: Yes, you are right, it's the same.

**Pull request merged**